compiler/cgen.nim

```diff
@@ -48,8 +48,8 @@ proc addForwardedProc(m: BModule, prc: PSym) =
   m.g.forwardedProcs.add(prc)
 
 proc findPendingModule(m: BModule, s: PSym): BModule =
-  let ms = s.itemId.module  #getModule(s)
-  result = m.g.modules[ms]
```

Comment:
`s.itemId.module` must contain the correct value. If not, that's where to patch it.